load_data.txt

```diff
@@ -0,0 +1,6 @@
+HANNAH KOSKE FELLOW Y
+BENJAMIN KAGIA FELLOW Y
```

Comment:
**Code Quality: Nitpick**

You should use a larger test file to enable you check for more edge cases